mybatis/runtime/src/main/java/io/quarkiverse/mybatis/runtime/ConfigurationBuilder.java

```diff
@@ -0,0 +1,13 @@
+package io.quarkiverse.mybatis.runtime;
+
+import org.apache.ibatis.session.Configuration;
+
+import com.baomidou.mybatisplus.core.MybatisConfiguration;
+
+public class ConfigurationBuilder {
+
+    public Configuration configuration() {
+        return new MybatisConfiguration();
```

Comment:
```MybatisConfiguration``` is from mybatis-plus library ? so it could be better to move to mybatis-plus directory